doc/Caliptra.md

```diff
@@ -233,116 +233,116 @@ In accordance with OCP Attestation specification [Reference 8](#ref-8), devices
 
 Operational security during the manufacturing process is critical, to ensure the DICE entropy is securely initialized, certified, and registered. Operational security avoids any pilfering of this asset by eavesdroppers. Operational security is outside the scope of this specification.
 
-# Threat Model
+# Threat model
```

Comment:
wouldn't we want headings to remain capitalized?

Reply:
No, current style is lowercase for headings.

Reply:
ack